### PR TITLE
Add missing qtbase5-private-dev package in INSTALL.md for Linux build

### PR DESCRIPTION
**Pull request by bagage**
I also removed commas so that one can copy the whole line directly to the terminal.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -73,8 +73,9 @@ PROJ.4 library and of CUPS are installed. For a Ubuntu or Debian system, install
 `libproj-dev` and `libcups2-dev`.
 
 For using the Qt libraries provided by the repositories, the following packages
-need to be installed:
-`qt5-default` (>= 5.2), `qttools5-dev`, `qttools5-dev-tools`, `libqt5sql5-sqlite`
+need to be installed. `qt5-default` needs at least version 5.2:
+
+`qt5-default` `qtbase5-private-dev` `qttools5-dev` `qttools5-dev-tools` `libqt5sql5-sqlite`
```
Comment (dg0yt) [inline INSTALL.md]: It is correct to add qtbase5-private-dev here, but the change of the wording about the version does not look like an improvement IMO.
(BTW the best documentation of build requirements are the control files on build.opensuse.org.)

**Comment by bagage** [comment]
Updated accordingly.

**Comment by bagage** [comment]
Up?

**Comment by dg0yt** [comment]
Well, focus was on other things recently, but I will pick it up now.

**Comment by dg0yt** [comment]
Merged. The reference got lost in another edit.

**Pull request closed without merging**